dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -172,6 +172,55 @@ def test_data_formats(self):
                     self.assertIsInstance(data, list)
                     self.assertIsInstance(data[0], str)
 
+    def test_set_header(self):
+        test_dir = os.path.join(test_root_path, 'data')
+        filename = 'csv/sparse-first-and-last-column-two-headers.txt'
+        filename = os.path.join(test_dir, filename)
+
+        # set bad header setting
+        options = dict(header=-2)
+        with self.assertRaisesRegex(ValueError,
+                                    '`header` must either be \'auto\' or an '
+                                    'integer > -1'):
```

Comment:
can you update the error message here to have the None (no header) value as well?

Reply:
Good catch